src/domain/journey/space/createSpace/plansTable/PlansTableDialog.tsx

```diff
@@ -0,0 +1,230 @@
+import DialogHeader from '../../../../../core/ui/dialog/DialogHeader';
+import RouterLink from '../../../../../core/ui/link/RouterLink';
+import { Caption, CaptionSmall, Text } from '../../../../../core/ui/typography';
+import GridContainer from '../../../../../core/ui/grid/GridContainer';
+import GridItem from '../../../../../core/ui/grid/GridItem';
+import { gutters } from '../../../../../core/ui/grid/utils';
+import WrapperMarkdown from '../../../../../core/ui/markdown/WrapperMarkdown';
+import Gutters from '../../../../../core/ui/grid/Gutters';
+import { usePlansTableQuery } from '../../../../../core/apollo/generated/apollo-hooks';
+import Loading from '../../../../../core/ui/loading/Loading';
+import SelectPlanButton from './SelectPlanButton';
+import { usePlanAvailability } from './usePlanAvailability';
+
+interface PlanTranslation {
+  name: string;
+  displayName: string;
+  priceDescription: string;
+  features: string[];
+  disclaimer: string;
+  disabledDisclaimer?: string;
+}
+
+const PlanName = styled('h1')(({ theme }) => ({
+  color: theme.palette.primary.main,
+  textAlign: 'center',
+  margin: 0,
+}));
+
+const Price = styled('h1')(({ theme }) => ({
+  color: theme.palette.primary.main,
+}));
+
+const lines = (theme: Theme) => `1px solid ${theme.palette.divider}`;
+
+interface PlansTableDialogProps {
+  open: boolean;
+  onClose: () => void;
+  onSelectPlan: (planName: string) => void;
+}
+
+const PlansTableDialog = ({ open, onClose, onSelectPlan }: PlansTableDialogProps) => {
+  const { t } = useTranslation();
+  const isSmall = useMediaQuery<Theme>(theme => theme.breakpoints.down('md'));
+  const isMobile = useMediaQuery<Theme>(theme => theme.breakpoints.down('sm'));
+
+  const { data, loading } = usePlansTableQuery({
+    skip: !open,
+  });
+
+  const { isFreePlanAvailable } = usePlanAvailability({ skip: !open });
+
+  const plansData = useMemo(
+    () => data?.platform.licensing.plans.filter(plan => plan.enabled).sort((a, b) => a.sortOrder - b.sortOrder) ?? [],
+    [data]
+  );
+
+  const [freeTrialDialogOpen, setFreeTrialDialogOpen] = useState<boolean>(false);
+  const [getStartedDialogOpen, setGetStartedDialogOpen] = useState<boolean>(false);
+  // Keep the first plan clicked here to be used in case in the Confirmation Dialog there is no planId
+  const [planSelected, setPlanSelected] = useState<string>();
+
+  useEffect(() => {
+    // Close any confirmation dialog if the main dialog is closed
+    if (!open && freeTrialDialogOpen) {
+      setFreeTrialDialogOpen(false);
+    }
+    if (!open && getStartedDialogOpen) {
+      setGetStartedDialogOpen(false);
+    }
+  }, [open]);
+
+  const planTranslations: Record<string, PlanTranslation> = t('plansTable.plans', { returnObjects: true }).reduce(
+    (acc, plan: PlanTranslation) => ({ ...acc, [plan.name]: plan }),
+    {}
+  );
+
+  const handlePlanClick = ({ name, isFree }: { name: string; isFree: boolean }) => {
+    setPlanSelected(name);
+    if (isFree) {
```

Comment:
I think here we need to check whether we have a plan with 'tiralEnabled'. If not, we should proceed directly with the creation of a Free plan (no trial dialog).